### ISSUE TITLE
SSTV modulator/demodulator

### ISSUE THREAD

**Issue by AsciiWolf** [f4exb/sdrangel#1406]
It would be great if SDRangel had a slow-scan television modulator and demodulator. Especially on Linux where there aren't many applications for SSTV.

**Comment by f4exb** [comment]
What about QSSTV: https://github.com/xdsopl/qsstv I saw your name there 😄

**Comment by AsciiWolf** [comment]
That GitHub repo is sadly just an unofficial fork. The official development page [was here](http://users.telenet.be/on4qz/), but it looks like that the development has died and the page is not available anymore.

**Comment by f4exb** [comment]
Maybe the fork has still valuable code I did not look into details but it is Qt based so it could be a good start. There are many SSTV standards and I do not have a lot of samples at hand so it is interesting to rely on proven code.

**Comment by f4exb** [comment]
I'll try to build and play with QSSTV to see how it looks like. May be useful to get some test signals also.

**Comment by srcejon** [comment]
I've just stuck a PD120 SSTV capture from the ISS here: http://www.sdrangel.org/iq/sstv.zip

Bit of a drop-out in signal strength in the middile, but strong at the start.

**Comment by sarvet** [comment]
Two resources I've found: 
https://github.com/ON4QZ/QSSTV
https://www.qsl.net/on4qz/
According to [this Post on Reddit](https://www.reddit.com/r/amateurradio/comments/ybzolm/anyone_know_what_happened_to_qsstv/):  "The provider Telenet has unilaterally stopped the Web service. Not even giving me a chance to put a redirect. […]"

**Comment by AsciiWolf** [comment]
Yep, I forgot to mention that the QSSTV was moved to https://www.qsl.net/on4qz/, so there is no need to use a fork or a Debian archive anymore.

**Comment by pchmielewski1** [comment]
3 years later, still no sdrangel support for sstv, I have to do a workaround to do it another way, I know free software I can't require, I don't recommend, often restarts. unstable software.

**Comment by dineshalphaque** [comment]
+1 to this feature request. SSTV support using PD120, Robot36 and Robot72 would be great!

**Comment by srcejon** [comment]
> +1 to this feature request. SSTV support using PD120, Robot36 and Robot72 would be great!

Can you upload some IQ .wav files containing examples of the signals you want supported. Thanks.

**Comment by dineshalphaque** [comment]
This playlist of audio captured for the different SSTV modulation formats could help. Robot36 is the most common, so would start with that before the others. 

https://youtube.com/playlist?list=PLhnEIZZXk7AOigIcDM0eobjqfmOtNsIqG&si=YSu1-jCjypQLPMGV

**Comment by AsciiWolf** [comment]
Or you can use [this example](https://en.wikipedia.org/wiki/File:SSTV_example_1.wav) (Scottie 1 mode) from Wikipedia.

More information about Scottie 1: https://radio.clubs.etsit.upm.es/blog/2019-08-10-sstv-scottie1-encoder/

[Another example](https://en.wikipedia.org/wiki/File:SSTV_sunset_audio.ogg) (Martin 1 mode) from Wikipedia.

More information about SSTV: https://en.wikipedia.org/wiki/Slow-scan_television

I would recommend implementing both Scottie 1 and Martin 1.

Both these SSTV modes are widely used: Scottie 1 mostly in USA, Martin 1 in Europe.

**Comment by srcejon** [comment]
Thanks. 

Would be useful to have an SSB IQ file rather than audio if anyone has one.

(I have a NFM/PD120 IQ file from an ISS broadcast.)
